client/my-sites/plugins/plugin-activate-toggle/index.jsx

```diff
@@ -146,8 +143,19 @@ PluginActivateToggle.defaultProps = {
 	disabled: false,
 };
 
-export default connect( null, {
-	recordGoogleEvent,
-	recordTracksEvent,
-	togglePluginActivation,
-} )( localize( PluginActivateToggle ) );
+export default connect(
+	( state, { site, plugin } ) => {
+		const pluginStatus = getStatusForPlugin( state, site.ID, plugin.id );
+		const inProgress =
```

Comment:
Would it make sense to move that to a new selector instead?

Reply:
Sure, that's a good call. I'll do that in a follow-up PR after this one and #47921 land.

Reply:
Addressed in #47951.